model/src/main/proto/spine_examples/shareaware/investment/purchase_events.proto

```diff
@@ -50,8 +50,8 @@ message SharesPurchased {
     // The ID of the purchased share.
     ShareId share = 3 [(required) = true];
 
-    // The quantity of the purchased shares.
-    int32 quantity = 4 [(required) = true];
+    // The number of available shares after purchase.
+    int32 shares_available = 4 [(required) = true];
```

Comment:
Consider renaming `shares_available` to `available_shares`. I think it's better sounds like `getAvailableShares` than `getSharesAvailable`.

Reply:
@Vladyslav-Kuksiuk `available_shares` would have to be a collection of something like `ShareId`.